core-java-modules/core-java-concurrency-advanced-4/src/main/java/com/baeldung/producerconsumer/Consumer.java

```diff
@@ -40,7 +43,7 @@ private void useMessage(Message message) {
             System.out.printf("[%s] Consuming Message. Id: %d, Data: %f\n", Thread.currentThread().getName(), message.getId(), message.getData());
 
             //Sleeping on random time to make it realistic
-            ThreadUtil.sleep((long) (message.getData() * 100));
+            // ThreadUtil.sleep((long) (message.getData() * 100));
```

Comment:
Rollback any changes used for testing requester's claim

Reply:
Done